### DISCUSSION TITLE
Send consolen / powershell output to liverly

### DISCUSSION THREAD

**Discussion by D1lema** [rocksdanister/lively#1855] [Ideas]
it is possible to send console output from PowerShell to the background?

**Comment by rocksdanister** [comment]
Not sure what exactly you are trying to do, if you can give a use case/example would help to understand better.

**Comment by D1lema** [comment]
I aim to move Powershell output to the desktop.
This is to display automatic processes, such as script start-up/execution, here.
I utilise Powershell for various automated tests. Occasionally, the Powershell window requires minimising to prevent any interference with the process and the output will 
not be visible.

**Reply by rocksdanister** [reply]
There are better ways to do this like making the powershell window bottom most instead of wallpaper.

I found this online:
https://www.autohotkey.com/board/topic/3262-send-window-to-bottom/

As to your original question I am not sure if powershell window can work as application wallpaper:
https://github.com/rocksdanister/lively/wiki/Application-Wallpaper
